src/test/java/org/verdictdb/core/querying/ola/AsyncAggJoinScaleTest.java

```diff
@@ -147,7 +147,7 @@ public void ScrambleTableTest() throws VerdictDBException,SQLException {
     stmt.execute("drop schema \"verdictdb_temp\" cascade;");
   }
 
-  @Test
+  //@Test
```

Comment:
Maybe uncomment? 

FYI, the channel should be set to the "getId()*1000" of the node that contains the placeholders. Simply using hard-coded integers (as in 5000 and 5001) is fine too.